### DISCUSSION TITLE
GPT-Pilot vs GPT-3

### DISCUSSION THREAD

**Discussion by lucas-0liveira** [Pythagora-io/gpt-pilot#150] [General]
How does GPT-Pilot differ from previous versions of the GPT-3 model, and in what kinds of applications can it be particularly useful?

**Comment by jajawunderbar** [comment]
It is only using! gpt3 or gpt4. Its not another version of gpt or something like this.

**Comment by Davidoluwatobi46** [comment]
Its just a software implementing GPT 3/4
Its not a new GPT version
